acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/Web3SignerProcessRunner.java

```diff
@@ -115,7 +115,7 @@ private void printOutput() {
         new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
       String line = in.readLine();
       while (line != null) {
-        PROCESS_LOG.info("Web3SignerProc: {}", line);
+        PROCESS_LOG.info(line);
```

Comment:
Is this confusing without the prefix? How do we know which log lines are from web3signer?

Reply:
They still have "subprocess" as the first entry in the line due to the name of the logger.